asm-to-pil/src/vm_to_constrained.rs

```diff
@@ -952,11 +948,11 @@ impl<T: FieldElement> ASMPILConverter<T> {
                 let prover_query_arms = free_value_query_arms.remove(reg).unwrap();
                 let prover_query = (!prover_query_arms.is_empty()).then_some({
                     FunctionDefinition::Query(Expression::LambdaExpression(LambdaExpression {
-                        params: vec!["i".to_string()],
+                        params: vec!["__i".to_string()],
```

Comment:
True, but we can always use `std::prover::eval(line)` or something. But yeah, let's keep it for block machines, and not mention it for VMs, because there, we really don't want anything to depend on explicit row numbers.